server/forge/github/parse_test.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	hookEvent  = "X-Github-Event"
+	hookEvent  = "X-GitHub-Event"
```

Comment:
I don't think that http headers are supposed to be case sensitive, so this change shouldn't matter, but it's notable.